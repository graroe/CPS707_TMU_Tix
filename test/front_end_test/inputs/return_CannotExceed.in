login
admin
add
CirqueDuSoleil
9988
return
CirqueDuSoleil
2
1
logout