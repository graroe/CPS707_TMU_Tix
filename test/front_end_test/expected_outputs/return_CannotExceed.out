Enter session type <sales or admin>: 
Admin login successful.
Enter event name: 
Current number of tickets: 10\nEnter number of tickets to add: 
9988 tickets added. New total: 9998
Enter event name: 
Current number of tickets: 9998\nEnter number of tickets to return: 
Total tickets cannot exceed 9999.\nEnter number of tickets to return: 
1 tickets returned. New total: 9999
Session terminated.